### PR TITLE
Create To Do List App Structure

### PR DESCRIPTION
**Pull request by trace-kadenyi**
Create to-do list app structure:
    - Set up `webpack `
    - Create the structure of the to-do-list app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+# Project Name: Webpack-Project: To-Do-List
+
+> Description of the project: A simple to-do-list app created using webpack.  
+
+
+## Built With
+
+- Major languages: HTML|CSS|JavaScript|Webpack
+
+
```
Comment (mahbubul14) [inline README.md:16]: - [x] In your `README` file, there is no content for the "Getting Started" section. Kindly add content for the getting started section. This should be an example of how you may give instructions on setting up your project locally. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @TracK92 ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comment under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (trace-kadenyi) [reply README.md:16]: Hello [mahbubul14](https://github.com/mahbubul14). Thanks for your review. However, shouldn't this be an optional suggestion? Why block my project over an omission in the README file which is not even mentioned in the project requirements? Unless I'm misunderstanding your statement.

Comment (mahbubul14) [reply README.md:16]: @TracK92, In the readme file, there is no getting started section. I think You should add a getting started section in your readme file.
You can check [this template ](https://github.com/microverseinc/readme-template)for the `README` to get an idea but remember to customize it to your project.

Comment (mahbubul14) [reply README.md:16]: @TracK92, The suggested change is a General requirement. You should always follow those general rules. It is not for a specific project. Here is the link for general [requirement.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

**Review by mugishasam123 — APPROVED** [review]
Hi @TracK92  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ No linter errors
✅ Descriptive Readme file

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (trace-kadenyi) [reply README.md:16]: [mahbubul14](https://github.com/mahbubul14) noted.

**Pull request merged**